rb/spec/integration/selenium/webdriver/spec_support/test_environment.rb

```diff
@@ -313,6 +315,15 @@ def random_port
         ensure
           sock.close
         end
+
+        def select_driver(opts)
```

Comment:
I think we can just alias chrome_beta_driver and chrome_driver, and just make the changes to browser version in #chrome_beta_options and #firefox_beta_options